src/react-imgix.jsx

```diff
@@ -91,14 +99,19 @@ function buildSrcSetPairFunction(url, aspectRatioDecimal = 0, fixedHeight) {
  */
 function buildDprSrcFunction(url, disableQualityByDPR, quality) {
   // Use quality if explicitly passed in -- either as an option or already in the url
-  if (disableQualityByDPR && quality) {
-    return dpr => `${url}&q=${quality}&dpr=${dpr} ${dpr}x`;
+  if (quality) {
+    const buildDprSrcWithQuality = (_, dpr) =>
```

Comment:
What I've gleaned from modern Node perf blog posts is that this could potentially perform even better if this is defined statically outside the `buildDprSrcFunction` function, because Node will automatically memoize/optimize the function.

Reply:
@frederickfogerty Mind sharing some of those blog posts? 😅